src/app/home/_component/mission_badge.tsx

```diff
@@ -0,0 +1,38 @@
+import { MoonIcon } from '@heroicons/react/20/solid';
+import { TicketIcon } from '@heroicons/react/20/solid';
+import { RocketLaunchIcon } from '@heroicons/react/20/solid';
+
+import Badge from '@/components/ui/badge';
+
+type Mission_type = 'DAILY' | 'WEEKLY' | 'SPECIAL';
```

Comment:
타입 이름은 카멜 케이스로 작성해주세요.

```suggestion
type MissionType = 'DAILY' | 'WEEKLY' | 'SPECIAL';
```